TsRandomizer/Randomisation/ItemLocationMap.cs

```diff
@@ -403,10 +403,10 @@ void AddPastItemLocations()
 		void AddPyramidItemLocations()
 		{
 			areaName = "Ancient Pyramid";
-			Add(new ItemKey(16, 14, 312, 192), "Why not it's right there", ItemProvider.Get(EItemType.MaxSand), LeftPyramid);
-			Add(new ItemKey(16, 3, 88, 192), "Conviction guarded room", ItemProvider.Get(EItemType.MaxHP), LeftPyramid);
+			Add(new ItemKey(16, 14, 312, 192), "Bottom of the Pit", ItemProvider.Get(EItemType.MaxSand), LeftPyramid); //more descriptive
+			Add(new ItemKey(16, 3, 88, 192), "Awkward Conviction room", ItemProvider.Get(EItemType.MaxHP), LeftPyramid); //honestly how else do you describe it
```

Comment:
This is the only room that has a check and a Conviction and I don't know what is awkward about it